src/database/initialization/application_form_field_to_json.sql

```diff
@@ -7,14 +7,13 @@ BEGIN
   INTO base_field_json
   FROM base_fields
   WHERE base_fields.id = application_form_field.base_field_id;
-
+  
```

Comment:
Watch out for inadvertent whitespace changes!

```suggestion

```

Reply:
good catch, thank you @jasonaowen !